src/scoring.jl

```diff
@@ -51,18 +55,16 @@ using ..NurseSchedules:
 
 function score(
     schedule_shifts::ScheduleShifts,
-    month_info::Dict{String,Any},
-    workers_info::Dict{String,Any};
-    return_errors::Bool = false,
+    schedule::Schedule;
+    return_errors::Bool = false
 )::ScoringResultOrPenalty
-    workers, shifts = schedule_shifts
     score_res = ScoringResult((0, []))
 
-    score_res += ck_workers_presence(schedule_shifts, month_info, workers_info)
+    score_res += ck_workers_presence(schedule_shifts, schedule)
```

Comment:
month_info i workers_info zmieniles w schedule? moze w schedule_info?

Reply:
Jak patrzę to już jest to na masterze, tylko brancha robiłem przed mergem poprzedniego taska i dlatego może to widzieć jako zmianę.

Po lewej task 115, po prawej aktualny master
![image](https://user-images.githubusercontent.com/10065832/104009203-eef11800-51aa-11eb-8774-4ee21407a04a.png)

Reply:
Tak czy siak, może _schedule_info_ :)